db/data_migrations/20201126094009_set_types_for_complaints.rb

```diff
@@ -0,0 +1,10 @@
+class SetTypesForComplaints < ActiveRecord::DataMigration
+  def up
+    kases = Case::Base.offender_sar_complaint.all
```

Comment:
Is this best as a migration or a rake task? Just thinking about the sequencing of the migration. Maybe I have the wrong end of the stick, but could you explain your reasoning to me :)

Reply:
You're quite right - I did consider making it a rake task but we never remember to run them. Happy to change if you like?